### PR TITLE
AEA-909

### PR DESCRIPTION
**Pull request by harryhmc**

### CODE REVIEW

File: tests/e2e/pact/specs/eps-convert.spec.ts [hunk 1]
```diff
@@ -56,6 +59,41 @@ jestpact.pactWith(
...
+
+      test.each(errorCases)("should receive expected error code in response to %s message", async (desc: string, request: Bundle, response: string, responseMatcher: string, statusCode: string) => {
+        const regex = new RegExp(responseMatcher)
+        const isMatch = regex.test(response)
+        expect(isMatch).toBe(true)
+
+        const requestStr = LosslessJson.stringify(request)
+        const requestJson = JSON.parse(requestStr)
+
+        const interaction = {
```
Comment (dan-jones-iw) [inline tests/e2e/pact/specs/eps-convert.spec.ts:71]: reckon we can remove most of the duplication over these interaction objects?

**Review by ri-ch — COMMENTED** [review]
Can we split this PR up so that it either contains only the test prescriptions, or only the code changes?

**Review by mfjarvis — APPROVED** [review]

**Pull request merged**